DataCleaning/Combine_data.ipynb

```diff
@@ -132,7 +152,42 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
+
+   "execution_count": 8,
+   "id": "4b0901dd-367e-41d9-9ab4-70b6076f2aff",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "# Group by 'ASXCode' and filter out groups with only one unique year\n",
+    "combined_df = combined_df.groupby('ASX Code').filter(lambda x: x['Year'].nunique() > 1)"
```

Comment:
If we have couple of years data for an ASX code will that be worth keeping? 
For a stock being available in 2014 till say 2017 in our dataset and then it disappear, are they worth keeping?